third_party/move/move-compiler-v2/tests/ability-check/v1-typing/module_call_constraints_not_satisfied.exp

```diff
@@ -11,16 +11,11 @@ error: type `S` is missing required ability `key` (type was inferred)
    │
    = required by instantiating type parameter `R:key` of function `both`
 
-error: type `Coin` is missing required ability `copy` (type was inferred)
-   ┌─ tests/ability-check/v1-typing/module_call_constraints_not_satisfied.move:29:9
+error: constraint `integer` does not have required ability `key`
```

Comment:
The original error is shadowed by the new error.

Reply:
Can you add a variant of this test that shows the original error is still there if this one is fixed?

Reply:
Added a test in the same file